roles/client/tasks/main.yml

```diff
@@ -22,15 +22,22 @@
   retries: 5
   when: ursula_os == 'rhel' and openstack_install_method == 'distro'
 
+- name: get constraints file
+  get_url:
+    url: "{{ openstack.upper_constraints }}"
+    dest: /tmp/constraints.txt
```

Comment:
client role comes before openstack server roles, there could be no /opt/stack directory. Also do we need to keep constraints file in /opt/stack for openstack **client**? I thought /opt/stack is for openstack servers.